lib/serverengine/process_manager.rb

```diff
@@ -146,9 +146,12 @@ def spawn(*args)
       rpipe, wpipe = new_pipe_pair
 
       begin
-        options[[wpipe.fileno]] = wpipe
-        if @enable_heartbeat
-          env['SERVERENGINE_HEARTBEAT_PIPE'] = wpipe.fileno.to_s
+
+        unless $platformwin
```

Comment:
Does this change mean heatbeat pipe doesn't work on Windows?
I concern about this because programs running as a child process need to add `if ENV.has_key?("SERVERENGINE_HEATBEAT_PIPE")` code to support Windows platform.